src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -125,9 +125,30 @@ public static class System
                 // public static readonly string System = "system";
                 public static readonly string TeamProject = "system.teamproject";
                 // back compat variable, do not document
+                public const string TeamProjectId = "system.teamProjectId";
                 public static readonly string TFServerUrl = "system.TeamFoundationServerUri";
                 public static readonly string PreferGit = "system.prefergit";
             }
+
+            public static class Release
+            {
+                //
+                // Keep alphabetical
+                //
+                public static readonly string AttemptNumber = "release.attemptNumber";
+                public static readonly string ReleaseDefinitionName = "release.definitionName";
+                public static readonly string ReleaseEnvironmentName = "release.environmentName";
+                public static readonly string ReleaseEnvironmentUri = "release.environmentUri";
+                public static readonly string ReleaseDescription = "release.releaseDescription";
+                public static readonly string ReleaseId = "release.releaseId";
+                public static readonly string ReleaseName = "release.releaseName";
+                public static readonly string ReleaseRequestedForId = "release.requestedForId";
+                public static readonly string ReleaseUri = "release.releaseUri";
+                public static readonly string ReleaseWebUrl = "release.releaseWebUrl";
+                public static readonly string RequestorId = "release.requestedFor";
+                public static readonly string SkipArtifactsDownload = "release.skipartifactsDownload";
+                public static readonly string Status = "status";
```

Comment:
Yes, these are variables. I have defined the constants like build and system variables here.
IMO adding a get oneliner for variables like the below

`public bool? Build_SyncSources { get { return GetBoolean(Constants.Variables.Build.SyncSources); } }`

isn't much useful. I would directly use
Variables.Get(Constants.Variables.Build.SyncSources) ?? true;

Or did I misunderstood your comment? :)